clusters/app.ci/openshift-user-workload-monitoring/mixins/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+SHELL=/usr/bin/env bash -o pipefail
+
+dashboards = cluster-pool
+
+all: $(dashboards) ci-alerts_prometheusrule.yaml alertmanager-user-workload-secret_template.yaml
+
+$(dashboards):
+	@echo "Generating $@.json ..."
+	mkdir -p tmp
```

Comment:
maybe change this to $TMP?

Reply:
Thought about it.
I sometimes need to check the file for debugging and it is easier to see the file in the current folder in the editor.